lib/vast/Dialect/HighLevel/Transforms/HLLowerTypes.cpp

```diff
@@ -438,50 +440,26 @@ namespace vast::hl
         using Base = LowerHLTypePatternBase;
         using Base::Base;
 
-
-        // This is a very random list that is pulled from sources of mlir of version
-        // llvm-14.
-        static bool is_fn_attr(mlir::NamedAttribute attr)
-        {
-            auto name = attr.getName();
-            if (name == mlir::SymbolTable::getSymbolAttrName() ||
-                name == mlir::FunctionOpInterface::getTypeAttrName() ||
-                name == "std.varargs")
-            {
-                return false;
-            }
-
-            // Expected the checks will expand.
-            return true;
-        }
-
-        auto lower_fn_attrs(FuncOp fn) const
-        {
-            return this->Base::lower_attrs(fn->getAttrs(), is_fn_attr);
-        }
-
         using attrs_t = mlir::SmallVector< mlir::Attribute, 4 >;
         using maybe_attrs_t = std::optional< attrs_t >;
         using signature_conversion_t = mlir::TypeConverter::SignatureConversion;
 
-        attrs_t lower_args_attrs(FuncOp fn, mlir::ArrayAttr arg_dict,
-                                 const signature_conversion_t &signature) const
+        auto lower_attrs(FuncOp fn, mlir::ArrayAttr attrs) const
         {
-            attrs_t new_attrs;
-            for (std::size_t i = 0; i < fn.getNumArguments(); ++i)
+            std::vector< mlir::DictionaryAttr > partials;
+
+            VAST_CHECK(!attrs, "Lowering of function attributes is WIP.");
```

Comment:
Is this still supposed to be here? Seems like problematic stuff should be caught by the assert that's few lines further.